gramps/gen/plug/utils.py

```diff
@@ -333,12 +333,7 @@ def load_addon_file(path, callback=None):
     """
     import tarfile
 
-    if (
-        path.startswith("http://")
-        or path.startswith("https://")
-        or path.startswith("ftp://")
-        or path.startswith("file://")
-    ):
+    if path.startswith(("http://", "https://", "ftp://", "file://")):
```

Comment:
```suggestion
    if path.lower().startswith(("http://", "https://", "ftp://", "file://")):
```
The protocol in a URI is case-insensitive. See https://www.rfc-editor.org/rfc/rfc3986#section-3.1